rpc/src/v1/helpers/subscribers.rs

```diff
@@ -36,8 +36,9 @@ impl str::FromStr for Id {
 	}
 }
 impl Id {
+	// FIXME: [niklasad1] replace `format` with `serde_json::to_string_pretty`
```

Comment:
Should make this an issue?

Reply:
Yes, when I benched it `serde::deser` was 10 times faster than `format!`